helm/g2p-sandbox-fynarfin-SIT/values.yaml

```diff
@@ -157,7 +157,7 @@ ph-ee-g2psandbox:
       enabled: true
       tenants: "rhino,gorilla"
       aws:
-        region: "us-east-2"
+        region: "ap-south-1"
```

Comment:
This change cannot be applied as the S3 bucker is not created yet. please verify